src/System.Runtime/tests/System/EnumTests.cs

```diff
@@ -62,6 +62,7 @@ public static IEnumerable<object[]> Parse_TestData()
             yield return new object[] { "1", false, UInt64Enum.One };
             yield return new object[] { "5", false, (UInt64Enum)5 };
 
+#if netcoreapp
```

Comment:
it is because missing APIs. do you want me split the test in this PR or we can do it later?